### ISSUE TITLE
Impossible to create wmi filter without description (or with empty description)

### ISSUE THREAD

**Issue by Yannik** [FriedrichWeinmann/GPWmiFilter#9]
Hi,

in the mmc, it's easily possible to create a WMI Filter without a description.

Unfortunately, it fails with your module:

No `-Description` given
```
PS C:\Users\Administrator\Downloads> New-GPWmiFilter -Name "Not_Core_OS" -Filter "Select * from CIM_DataFile where Name='C:\\Windows\\explorer.exe'"
WARNING: [23:49:20][New-GPWmiFilter] Failed to: <String Key not found: GPWmiFilter.New-GPWmiFilter.CreatingFilter> | The server is unwilling to process the request
```

`-Description ''`
``` 
PS C:\Users\Administrator\Downloads> New-GPWmiFilter -Name "Not_Core_OS" -Filter "Select * from CIM_DataFile where Name='C:\\Windows\\explorer.exe'" -Description ''
WARNING: [23:49:37][New-GPWmiFilter] Failed to: <String Key not found: GPWmiFilter.New-GPWmiFilter.CreatingFilter> | The server is unwilling to process the request
```

**Comment by FriedrichWeinmann** [comment]
Heya Yannik,
bug has been resolved in 1.0.5

**Comment by Yannik** [comment]
Great, thank you!
